public/src/js/models/config/collection.js

```diff
@@ -20,11 +19,9 @@ export default class ConfigCollection extends DropTarget {
 
         this.id = opts.id;
 
-        const defaults = vars.model.state().defaults;
-
         this.parents = ko.observableArray(findParents(opts.id));
         this.userVisibilities = CONST.userVisibilities;
-        this.availableTerritories = getAvailableTerritories(defaults);
+        this.availableTerritories = [];
```

Comment:
```suggestion
        this.availableTerritories = defaults ?? [];
```

Totally agree that we should remove the silly function, but this removes the defaults altogether, which means we can no longer target territories:

<img width="595" alt="Screenshot 2022-12-12 at 09 21 01" src="https://user-images.githubusercontent.com/7767575/207008521-cf92c967-1c68-40c5-9bf3-266eae1cddfe.png">

Unsure if this was a response to breaking tests – wonder if they're need in a fixture somewhere.

Reply:
Having looked into this further you are completely right! 

`defaults` come from the server on calling /defaults 

Tests were failing because defaults will be undefined when the tests run so adding an extra check to fix that.

All should be well

Reply:
Ace – just tested, territories are now there.